tests/test_modeling_bart.py

```diff
@@ -17,7 +17,7 @@
 import tempfile
 import unittest
 
-from transformers import is_torch_available
+from transformers import is_torch_available  # dont add here, add under is_torch_available
```

Comment:
extraneous comment?

Reply:
It's an attempt to avoid breaking the tf only tests (which I have done like 10 times) by adding imports on the top level.